components/_impl/workers/subprocess_worker.py

```diff
@@ -372,4 +378,5 @@ def __del__(self) -> None:
         self._stderr_f.close()
 
         # Finally, make sure we don't leak any files.
```

Comment:
nit: could update this comment to reflect the change.

Reply:
Thanks! Addressed